carddrawer/src/main/java/com/meli/android/carddrawer/ViewHelper.kt

```diff
@@ -0,0 +1,48 @@
+package com.meli.android.carddrawer
+
+import android.content.Context
+import android.graphics.drawable.GradientDrawable
+import android.view.View
+import androidx.core.content.ContextCompat
+import com.meli.android.carddrawer.ColorUtils.safeParcelColor
+
+internal object ViewHelper {
+
+    @JvmStatic
+    fun getGradientDrawable(context: Context,
+        gradientColors: List<String?>?): GradientDrawable {
+        val gradientDrawable = ContextCompat.getDrawable(context, R.drawable.card_drawer_gradient) as GradientDrawable
+        return if (gradientColors == null || gradientColors.isEmpty()) {
```

Comment:
no hay una ktx para is empty que chequea null tambien? no podemos usar takeIf o algo en vez del if?